src/integrationTest/resources/db/migration/V1_9999__integrationTestData.sql

```diff
@@ -0,0 +1 @@
+INSERT INTO audit VALUES (1, 2, 3, 4, '2023-06-13T08:13:09.688537759Z', 'application_server', 'additional_data');
```

Comment:
What's wrong with inserting at 1? is there an issue where tests might interfere with each other?

Reply:
you haven't reset the sequence, so the nextval of the sequence is still sitting at 1, so the next time the code tries to insert into it, it will try to insert with ID 1 also. And will therefore cause a unique constraint violation.

Reply:
ACCEPT. its not issue now because we only need 1 insert to test but changed the id to be a high value.